reposilite-backend/src/main/kotlin/com/reposilite/console/application/ConsolePlugin.kt

```diff
@@ -66,14 +82,19 @@ internal class ConsolePlugin : ReposilitePlugin() {
             event.config.router.mount {
                 it.ws(
                     "/api/console/sock",
-                    CliEndpoint(
+                    ConsoleWebSocketHandler(
                         journalist = reposilite().journalist,
                         accessTokenFacade = facade(),
                         authenticationFacade = facade(),
                         consoleFacade = consoleFacade,
                         forwardedIp = sharedConfigurationFacade.getDomainSettings<WebSettings>().computed { it.forwardedIp }
                     )
                 )
+                it.sse(
+                    // TODO: does this need better endpoint name?
+                    "/api/console/log",
+                    client
```

Comment:
I think `"/api/console/log"` is okay, in the end this endpoint only returns live logs from the cli now